lib/src/main/java/org/team100/lib/config/NotePoseDetector.java

```diff
@@ -134,7 +130,7 @@ public Pose2d fieldRelativePose2d() {
         switch (Identity.instance) {
             case BETA_BOT:
             case COMP_BOT:
-                return m_swerve.getPose().transformBy(robotRelativeTransform2d());
+                return m_swervePose.getEstimatedPosition().transformBy(robotRelativeTransform2d());
```

Comment:
i'm confused about what the "rotation" of this object represents.  the camera isn't sensing rotation.  what is being measured by the rotational component of this pose?